services/website-api/src/controller/payment/PaymentController.ts

```diff
@@ -0,0 +1,64 @@
+import * as express from 'express';
+import { PaymentService } from '@donategifts/payment';
+import {
+	/* IUser, */ IDonationHook,

```

Comment:
remove the commented stuff please

Reply:
It's used for the onSuccess route (which is also commented out, I didn't remove it because I wasn't sure if we were going to use it. It depends on how things are handled on frontend side).

The commented stuff on PaymentServices goes with this route :)